acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multikeysigner/HashicorpBasedTomlLoadingAcceptanceTest.java

```diff
@@ -38,14 +38,14 @@
   static final String HASHICORP_ETHEREUM_ADDRESS = "0x" + FILENAME;
 
   private static HashicorpVaultDocker hashicorpVaultDocker;
-  private static final HashicorpNode hashicorpNode = new HashicorpNode();
+  private static final HashicorpVault HASHICORP_VAULT = new HashicorpVault();
```

Comment:
Do we consider this to be a constant? It is effectively a constant because it is static, but other static fields not using primitives don't seemed be capitalised like this.

Reply:
So - the refactor made that change - but if something is static and final - its a constant - should it be capitalised ... no idea.